src/wtstat.f

```diff
@@ -1,9 +1,10 @@
 C> @file
-C> @brief Update the status of a system file with respect to the BUFRLIB software.
+C> @brief Open a BUFR file, or update file status.
 C>
```

Comment:
OK, but this function says:

C> This subroutine can be used to connect or disconnect a specified
C> Fortran logical unit number to/from the BUFRLIB software, and it
C> can also be used to set or reset the internal message status
C> associated with that logical unit number.

Doesn't that mean it does an open like openbf()?

Reply:
In other words, in this case "connect or disconnect" means it officially registers the presence of this logical unit within its internal file ID array.  But again, there's no "open" or "close" ever going on in this routine.

Reply:
I understand it can be a bit confusing, but openbf internally calls status (to get a new file ID) and then wtstat (to actually record the new file ID).  But then wtstat also gets called later by other routines when, e.g. a new message is opened or closed for that file ID.

So all wtstat ever does is just update internal information (i.e. status information ;-) corresponding to a file ID.  It never actually opens a file.

Reply:
OK, got it, let me see if I can say it better...

Reply:
Maybe in the brief description of the function we should also indicate whether it is internal? In other words, not intended for the user to call directly?

Reply:
OK, I have rephrased. Take a look.

Reply:
Correct, wtstat is only intended to be used internally and should never be called directly by a user.